### DISCUSSION TITLE
Teleport player to Spawn Location after Join server

### DISCUSSION THREAD

**Discussion by IceBreathNetDevelopment** [Minestom/Minestom#1346] [Help] [answered]
Hello i want to teleport a Player who is joining on the Minestom server but it is not teleporting....

The loc Pos is right and nut null and the cordinates also right, but it is not teleporting.

    public void onHandle(PlayerSpawnEvent playerSpawnEvent) {
        var player = playerSpawnEvent.getPlayer();
       
        var loc = MinestomBootstrap.getInstance().getUtils().getLocation(Locations.LOCATION_PLAYER_SPAWN).position().toPos();
        player.teleport(loc);
{
   
   
   PS: sorry for my bad english

**Comment by KrystilizeNevaDies** [comment, marked as answer]
Player#setRespawnPoint is probably what you are looking for here.
What's happening is you are teleporting the player before the event has processed. And after the event has processed, the player will continue to spawn where it was told to spawn previously (Usually it's respawn point).
